core/src/main/java/com/linecorp/armeria/server/DefaultServerConfig.java

```diff
@@ -112,6 +112,7 @@ final class DefaultServerConfig implements ServerConfig {
     private final Http1HeaderNaming http1HeaderNaming;
     private final DependencyInjector dependencyInjector;
     private final List<ShutdownSupport> shutdownSupports;
+    private final Duration unLoggedExceptionReportInterval;
```

Comment:
`unlogged`

Reply:
All `UnLog` is changed into `Unlog`. I think unlog is the right word 😃 .